toolkit/Makefile

```diff
@@ -120,6 +122,21 @@ ifeq ($(USE_PREVIEW_REPO),y)
    endif
 endif
 
+ifeq ($(USE_EXTENDED_REPO),y)
+   PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/extended/$(build_arch)
+   PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/extended/debuginfo/$(build_arch)
+   SRPM_URL_LIST      += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/extended/srpms
+   ifneq ($(wildcard $(EXTENDED_REPO)),)
+      override REPO_LIST += $(EXTENDED_REPO)
+   else
+      $(warning )
+      $(warning ######################### WARNING #########################)
+      $(warning 'USE_EXTENDED_REPO=y' set but '$(EXTENDED_REPO)' is missing. Regenerate toolkit's 'repos' directory. Remove 'USE_EXTENDED_REPO' for core builds.)
+      $(warning ######################### WARNING #########################)
+      $(warning )
+   endif
+endif
+
```

Comment:
These changes look unrelated to this PR. Also, GitHub is saying the PR "wants to merge 197 commits into microsoft:main". Can you rebase your changes on top of the latest main branch revision?

Reply:
They're not directly related to this PR, but building openblas takes a few hours, so I used this addition to the Makefile to skip that. I left it in this PR because I thought it might be useful for anyone else using dependencies from extended that take a long time to build. Working on the rebasing.

Reply:
I really like the change! We should separate it out from the packaging changes. One half of this PR is packaging bugfixes, the other half is a toolkit change to improve developer experience. Separate PRs allow us to have a more nuanced and focused conversation about each topic.

Reply:
Gotcha! I'll get on that then :)